inclusive-design/wecount.inclusivedesign.ca
# Side nav enhance input size (resolves #296)

## Description
The size of the side menu nav fonts remains the same when enhance inputs is turned on.


## Steps to test

1. Open to deploy preview
2. Go to About page
3. Turn on UIO enhance inputs
4. Observe side menu nav item's `font-size`

**Expected behavior:** 
`font-size` of side menu nav component should remain the same when enhance inputs is turned on.

## Additional information
N/A


## Related issues
closes #296
